### PR TITLE
Enoriega/update img name

### PR DESCRIPTION
**Pull request by enoriega**
Update the docker compose file to point to the official image

### CODE REVIEW

**Comment by enoriega** [comment]
Still a draft until I can verify that the compose file works well with the latest image

**Comment by enoriega** [comment]
@myedibleenso Tested and worked

File: end-to-end-rest/docker-compose.yml [hunk 1]
```diff
@@ -9,13 +9,13 @@ services:
       - "9010:8000"
 
   skema-tr:
-    image: enoriega87/skema-webapp
+    image: lumai/askem-skema-text-reading
     ports:
-      - "9005:9000"
+      - "9000:9000"
     environment:
       secret: 8c2fb5a58ae8d9ec6f7065a25c35aac2
```
Comment (myedibleenso) [inline end-to-end-rest/docker-compose.yml]: This should now be `APPLICATION_SECRET`

File: end-to-end-rest/docker-compose.yml [hunk 2]
```diff
@@ -9,13 +9,13 @@ services:
...
       SKEMA_HOSTNAME: skema.clulab.org
-      _JAVA_OPTIONS: -Xmx20g -Xms20g -Dfile.encoding=UTF-8
+      _JAVA_OPTIONS: -Xmx10g -Xms10g -Dfile.encoding=UTF-8
```
Comment (myedibleenso) [inline end-to-end-rest/docker-compose.yml:18]: Is this the minimum RAM needed, @enoriega ?

**Review by myedibleenso — CHANGES_REQUESTED** [review]
Sorry, @enoriega .  I forgot all about this 😅 !

File: end-to-end-rest/docker-compose.yml (hunk 2 above)
Comment (enoriega) [reply end-to-end-rest/docker-compose.yml:18]: Yes.  Hopefully we will be able to bring it down soon

File: end-to-end-rest/docker-compose.yml (hunk 1 above)
Comment (enoriega) [reply end-to-end-rest/docker-compose.yml]: Changed

File: end-to-end-rest/docker-compose.yml [hunk 3]
```diff
@@ -9,13 +9,13 @@ services:
       - "9010:8000"
 
   skema-tr:
-    image: enoriega87/skema-webapp
+    image: lumai/askem-skema-text-reading
```
Comment (myedibleenso) [inline end-to-end-rest/docker-compose.yml]: @enoriega , should we explicitly add the `:latest` tag as well?  What do you think?

**Review by myedibleenso — APPROVED** [review]

File: end-to-end-rest/docker-compose.yml (hunk 3 above)
Comment (enoriega) [reply end-to-end-rest/docker-compose.yml]: Agreed

**Pull request merged**